charts/steward/values.yaml

```diff
@@ -94,5 +94,5 @@ hooks:
   images:
     kubectl:
       repository: docker.io/bitnami/kubectl
-      tag: "1.14.3"
+      tag: "1.17"
```

Comment:
Potential side effects expected?

Reply:
It's just used in the Helm hook to apply the CRDs:

    cat "$CRD" | kubectl apply -f -

This basic operation should still work as before.